articles/governance/policy/how-to/remediate-resources.md

```diff
@@ -88,6 +88,8 @@ is selected.
 
 1. Specify the location at which the managed identity is to be located.
 
+1. You do not need to select a Scope for System-Assigned Managed Identity, as the scope will be inherited from the assignment scope. 
```

Comment:
do not should be changed to don't
Scope and System-Assigned Managed Identity should be lowercase.

Reply:
Adjusted